apps/models/app_types/custom/custom.py

```diff
@@ -1,14 +1,40 @@
+import regex as re
+from django.core.exceptions import ValidationError
```

Comment:
```suggestion
import regex as re

from django.core.exceptions import ValidationError
```

This should be caught by isort I think

Reply:
Thank you. Addressed it.